themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
+
+There are three places in the VuFind configuration that you can change to see your custom theme in action.
+
+The first and most straight-forward is by changing `config.ini > [Site] > theme`. This will change the default theme for everyone who comes to your site and is the setting you should change for production.
+
+The two more development-oriented options are below `theme`: `alternate_themes` and `selectable_themes`. They both take comma-separated lists of themes in the format of `name1:theme1,name2:theme2`. Changing the theme via either of these two options sets a cookie that will keep your theme choice over the course of your session.
+
+Setting up themes under `alternate_themes` allows you to add a `ui=name` parameter to the end of your URL to see that theme. For example, the default configuration of `custom:custom_tmpl` would cause `https://vufind.org/?ui=custom` to show the theme described in the directory `custom_tmpl`.
```

Comment:
I'm assuming that we really don't want users editing this example theme -- it should be a template that they can copy. Perhaps we should actually treat this the same way that we treat custom code modules, and adjust install.php so that a user can request the creation of a theme, and the installer will actually copy the template theme to a new directory (user's name of choice). The unfortunate thing about that idea is that config.ini doesn't exist yet when install.php runs (it's created by the web-based installer later) so we can't easily turn on the user's custom theme by default without doing something a bit complicated.